api/controllers/v0.2/createPatron.js

```diff
@@ -163,73 +261,52 @@ function createPatron(req, res) {
-          headers: {
-            'Content-Type': 'application/json',
-            Authorization: `Bearer ${tokenResponse.data.access_token}`,
-          },
-        })
-          .then((response) => {
-            const modeledResponse =
-              modelResponse.patronCreator(response.data, response.status, patronData);
-            modelStreamPatron.transformPatronRequest(
-              req.body, modeledResponse // eslint-disable-line comma-dangle
-            )
-              .then((streamPatron) => {
-                streamPublish.streamPublish(
-                  process.env.PATRON_SCHEMA_NAME_V02,
-                  process.env.PATRON_STREAM_NAME_V02,
-                  streamPatron // eslint-disable-line comma-dangle
-                ).then((streamResponse) => { // eslint-disable-line no-unused-vars
-                  renderResponse(req, res, 201, modeledResponse);
-                  logger.debug('Published to stream successfully!', {routeTag: ROUTE_TAG});
-                }).catch((streamError) => {
-                  renderResponse(req, res, 201, streamError);
-                  logger.error(`Error publishing to stream: ${streamError}`, {routeTag: ROUTE_TAG});
-                });
-              })
-              .catch((error) => {
-                renderResponse(req, res, 201, modeledResponse);
-                logger.error(`Error creating patron: ${error}`, {routeTag: ROUTE_TAG});
-              });
-          })
-          .catch((response) => {
-            logger.error(
-              `status_code: ${response.response.status}, ` +
-              'type: "invalid-request", ' +
-              `message: "${response.message} from ILS.", ` +
-              `response: ${JSON.stringify(response.response.data)}`,
-              {routeTag: ROUTE_TAG} // eslint-disable-line comma-dangle
-            );
-
-            if (response.response && response.response.data) {
-              const responseObject = collectErrorResponseData(
-                response.response.status,
-                response.response.data.type,
-                response.response.data.detail,
-                response.response.data.title,
-                response.response.data.debug_message // eslint-disable-line comma-dangle
-              );
-
-              const statusCode = (responseObject.status) ? responseObject.status : 500;
-
-              renderResponse(
-                req,
-                res,
-                statusCode,
-                modelResponse.errorResponseData(responseObject) // eslint-disable-line comma-dangle
-              );
-            } else {
-              renderResponse(req, res, 500, modelResponse.errorResponseData(
-                collectErrorResponseData(null, '', '', '', '') // eslint-disable-line comma-dangle
-              ));
-            }
-          });
       });
   });
 }
 
-module.exports = {
-  createPatron,
-};
+if (process.env.NODE_ENV === 'test') {
```

Comment:
This is unconventional, but I wanted to export `validateEnvironmentAndRequest` and `validateEnvVariable` in the test environment so that I could test them.